<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mHpQtButton</class>
 <widget class="QDialog" name="mHpQtButton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>125</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>302</width>
    <height>125</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>302</width>
    <height>125</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Checkable Qt Button State</string>
  </property>
  <widget class="QRadioButton" name="RadChecked">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Checked</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="RadUnchecked">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Unchecked</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RadChange">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Change state</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>State value :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EdtState">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
